cscs-checks/microbenchmarks/stream/stream.py

```diff
@@ -56,30 +58,30 @@ def __init__(self):
         }
         self.stream_bw_reference = {
             'PrgEnv-cray': {
```

Comment:
You should merge with master to take the `PrgEnv-cray_classic` prgenv and this to the supported systems as well.